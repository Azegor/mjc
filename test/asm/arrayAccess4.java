class Foobar {
  public static void main(String[] args) {
    int[] a = new int[10];
    int[] b = new int[1];

    a[9] = 1337;
    System.out.println(a[9]);
    b[0] = a[9] - 1;
    System.out.println(b[0]);
  }
}
