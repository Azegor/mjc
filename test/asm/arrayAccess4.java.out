1337
1336
